cinema/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 4.0.2 on 2022-05-03 13:41
```

Comment:
DO not modify existing migration file. Create a new one instead

Reply:
this is a new one!